doc/src/bond_cs.md

```diff
@@ -571,6 +571,19 @@ payload represented by a protocol reader `R`. The former is usually used by
 producers to initialize `bonded<T>` values, the latter is implicitly used 
 during deserialization.
 
+Standard implementations of `Bonded<T>` and `Bonded<T,R>` always use default
```

Comment:
- [ ]Minor: "The standard implementations always use default implementations of Serializer..."